.buildkite/scripts/dockerhub_publish.sh

```diff
@@ -4,15 +4,27 @@ set -euo pipefail
 
 source .buildkite/scripts/shared.sh
 version=$(buildkite-agent meta-data get "version")
-channel=$(buildkite-agent meta-data get release-channel)
 
 if is_fake_release; then
   # This overrides IMAGE_NAME in docker-base.sh
   export IMAGE_NAME="habitat/fakey-mc-fake-face-studio"
+else
+  export IMAGE_NAME="habitat/default-studio"
```

Comment:
Does this need to be exported now? It looks like you're using it pretty explicitly.

I think the general convention for things that are just local variables (as opposed to environment variables) is to have them be in lower-case, e.g., `image_name`.

It doesn't look like we're sourcing `docker-base.sh`, either, so we can take that comment out.